app/helpers/payment_details_helper.rb

```diff
@@ -45,7 +45,7 @@ def payment_details_definition_list_item(name, account)
   def stripe_form_data(intent)
     {
       stripePublishableKey: site_account.payment_gateway_options[:publishable_key],
-      setupIntentSecret: intent.client_secret,
+      setupIntentSecret: intent&.client_secret,
```

Comment:
I would remove this, I don't think it's good to load the stripe form when the backend data is not correct.

Reply:
but this is an error handler if things comes nil

Reply:
I still think this isn't necessary, because now a customer is being created on error, this won't be null.